bin/riak-bucket-exporter.js

```diff
@@ -111,7 +119,14 @@ function processKey(key, cb) {
     }
     fs.appendFileSync(program.file, JSON.stringify(out,null,'\t'));
     first=false;
-    return cb();
+    
+    if (deleteKeys) {
```

Comment:
i would then go and merge this, increase the version and repbulish it. If you want to you can also add the version bump to the pull request and also please add yourself to the authors list.